activesupport/lib/active_support/reloader.rb

```diff
@@ -17,25 +14,23 @@ module ActiveSupport
   #   +reload_classes_only_on_change+ is false, the class unload will
   #   have occurred after the work run, but before this callback.
   #
-  #   before_class_unload -- Run immediately before the classes are
-  #   unloaded.
-  #
-  #   after_class_unload -- Run immediately after the classes are
-  #   unloaded.
-  #
   class Reloader < ExecutionWrapper
     define_callbacks :prepare
 
     define_callbacks :class_unload
 
+    # A hook run once at application startup and every time the code is reloaded.
+    # Can be used to monkey patch reloaded constants.
```

Comment:
I prefer to keep this second line out. It can be useful for a lot of things, so better to no specify.